utilities/aap.py

```
@@ -0,0 +1,651 @@
+    Raises:
+        TimeoutError: If AWX API is not ready within timeout.
+    """
+    LOGGER.info(f"Waiting for AWX API to be fully ready at {awx_url} (timeout={timeout}s)")
+    deadline = time.time() + timeout
+
+    while time.time() < deadline:
+        try:
+            response = requests.get(
+                f"{awx_url}/api/v2/organizations/",
+                auth=(username, password),
+                verify=False,
```

Ruff (0.16.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)